gl/auth for session creation (#3)
cloudradar-monitoring/plexus: README.md

   ```bash
   $ curl -ks https://localhost:8080/session \
          -u support:1234 \
          -F id="helping-joe" \
          -F ttl=3600 \
          -F username=admin \
          -F password=foobaz |jq
   ```
   

>    - Basic authentication `-u` is required the credentials are part of `plexus.conf` file
>    under `PLEXUS_AUTH_USER` and `PLEXUS_AUTH_PASSWORD` keys.

Reviewer: This sounds a little weird, how about:
```suggestion
   - The configured credentials `PLEXUS_AUTH_USER` and `PLEXUS_AUTH_PASSWORD` are required when creating a session.
```

Author: Yeah, I agree. Writing is not my strong side.
